recipes/dgl/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "dgl" %}
+{% set version = "0.8.1" %}
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/dmlc/dgl
+  - git_rev: {{ version }}
```

```suggestion
  git_url: https://github.com/dmlc/dgl
  git_rev: {{ version }}
```

If you put it as a list it sees it as two sources, so I guess `git_rev` was being ignored!

---

Thanks!